hat/assets/js/apps/Iaso/domains/assignments/components/AssignmentsListTab.tsx

```diff
@@ -69,52 +83,96 @@ export const AssignmentsListTab: FunctionComponent<Props> = ({
     setParentSelected,
 }: Props) => {
     const columns = useColumns({
-        orgUnits,
+        orgUnits: orgUnits || [],
```

Comment:
type name should be upper case
```suggestion
        orgUnits: OrgUnits || [],
```